build/release/README.md

```diff
@@ -55,6 +55,47 @@ The tags allow for a progression of pre-releases such as:
 
 The release tags should be agreed on by the release team.
 
+## Bulk backporting master changes to a release branch
+During the short weeks leading up to release, the release branch will exist with betas released
+periodically. Betas will generally include all commits in master; that is, all commits in master are
+likely to be backported to the release branch. Here are instructions for how to bulk backport the
+master branch's commits to a release branch.
+
+Let's parameterize these instructions slightly
+```sh
+RELEASE_BRANCH='release-1.2'
+```
+
+The following views may be useful. Open them side-by-side.
+- **A :** `git log --format=oneline --no-merges --topo-order $RELEASE_BRANCH`
+- **B :** `git log --format=oneline --no-merges --topo-order master`
+
+Open a new window with the following view to list the commits which are part of the `master` branch
+and not part of the release branch. These will be in alphabetical sorted order rather than commit
+topological order.
+- **C :** `comm -1 -3 <(git log --format=format:'%s' --no-merges --topo-order release-1.2 | sort) <(git log --format=format:'%s' --no-merges --topo-order master | sort)`
+
+Open a fourth window to be our working window.
+1. `git co master`
+2. `git co -b working-branch-for-$RELEASE_BRANCH` to make a working branch equivalent with `master`
+3. `git rebase --interactive $RELEASE_BRANCH` to rebase `master` changes on *top* of the release branch
+4. Compare the output in **C** to the commits here. git is not perfect and sometimes tries to rebase
```

Comment:
If we need to check the commit list it's going to be error prone so I'm worried about this step. Did you find why git was trying to rebase those two commits for the backport earlier this week? Or you ended up just removing them from the commit history? It seems like something isn't right with our process if git is picking up extra commits.

Reply:
`git` is a magic black box to me. I don't know why? Did anyone backport commits in between bulk backports?